scalikejdbc-interpolation/src/test/scala/scalikejdbc/QueryInterfaceSpec.scala

```diff
@@ -152,6 +152,24 @@ class QueryInterfaceSpec extends FlatSpec with Matchers with DBSettings with SQL
 
         withSQL { delete.from(Product).where.in(pc.id, Seq(3, 4)) }.update.apply()
 
+        // batch insert with BatchParamsBuilder
+        {
+          val products: Seq[Product] = Seq(Product(3, Some("Coffee"), Price(90)), Product(3, Some("Coffee"), Price(200)))
```

Comment:
The test doesn't fail but it seems to be using the same ID - 3.

Reply:
Oops! thx